numpy/lib/histograms.py

```diff
@@ -230,7 +233,7 @@ def _get_outer_edges(a, range):
             'max must be larger than min in range parameter.')
```

Comment:
This case can move inside the `range is not None` case, I think.

Reply:
Yes, that makes sense. Since the autodetected range will never result in ```first_edge``` being greater than ```last_edge```.